Documentation: add insights documentation for admins (#4282)
mozilla/pontoon: documentation/docs/admin/insights.md

# Insights

Pontoon provides a global **Insights** dashboard for analyzing the health of localization communities and the quality of pretranslations across all locales and projects.

> The dashboard is restricted to staff and is available at [https://pontoon.mozilla.org/insights/](https://pontoon.mozilla.org/insights/) when logged in as an administrator.

Reviewer: ```suggestion
The dashboard is restricted to staff and is available at `/insights` when logged in as an administrator (e.g. [https://pontoon.mozilla.org/insights/](https://pontoon.mozilla.org/insights/) for the instance hosted by Mozilla).
```

## Community health activity

> This section presents a table of the monthly `Community health score` and its underlying metrics for each selected locale. For every team it shows the number of managers, translators, active contributors, all contributors and new signups, along with the number of enabled projects, project completion, and the resulting `Community health score`.

Reviewer: ```suggestion
This section presents a table of the monthly `Community health score` and its underlying metrics for each selected locale. For every team it shows the number of users in different roles — managers, translators, contributors — above a certain threshold of submitted translations, all contributors independently of the number of approved translations, and new signups, along with the number of enabled projects, project completion, and the resulting `Community health score`.
```

Each cell also displays the month-over-month change, making it possible to compare the current values against the previous month. Use the `Filter teams` search box to narrow the table down to specific locales.

### Edit locales

The `Edit Locales` button toggles a view where the user selects which locales are displayed in the dashboard and its charts. Move one or more locales into the target list to save preferred locales for display. If no locales are selected, the dashboard prompts the user to choose at least one before any data is shown. Press `Back` to toggle back to the dashboard view.

### Show scores

By default the table shows the raw value of each metric. Click the `Show scores` button to switch to the score view, which instead displays the individual component scores that add up to the `Community health score`. Click the button again (now labeled `Show default`) to return to the raw values.

## Community health score chart

> This chart plots the monthly `Community health scores` of all selected locales for the most recent 12 months, along with the 12-month average. Hover over a data point in the graph to see all selected locale scores for that month along with the 12-month average.

Reviewer: ```suggestion
This chart plots the monthly `Community health scores` of each selected locale for the most recent 12 months, along with the average of all selected locales. Hover over a data point in the graph to see each selected locale's score for that month along with the average.
```

I assume the average is based on the locales enabled?

Author: yes that's correct.
